packages/network/src/helpers/AddressTools.ts

```diff
@@ -0,0 +1,27 @@
+import ipaddr from 'ipaddr.js'
+
+// IPv4 private address ranges as specified by RFC 1918
+const IPv4PrivateRanges = [
+    '10.0.0.0/8',
+    '172.16.0.0/12',
+    '192.168.0.0/16',
+].map((a: string) => ipaddr.parseCIDR(a))
```

Comment:
same here: 
```suggestion
].map((a) => ipaddr.parseCIDR(a))
```
Up to you though.